src/cypress-api/commands.ts

```diff
@@ -0,0 +1,16 @@
+// @ts-nocheck
```

Comment:
I'm hoping to punt on making these Cypress files type-correct for now (until we create dedicated packages for Cypress and Playwright). To get them correct we'd need to install the Cypress types as a dependency, which means all Playwright users would have Cypress types installed.